### PR TITLE
Chrysler Pacifica Fingerprint

### PR DESCRIPTION
**Pull request by TheWizard1328**

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Please provide the dongleID (found in settings on your device) that this was taken from

**Comment by TheWizard1328** [comment]
My Dongle ID: 8a84e5027e568485

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by pd0wm** [comment]
Can you add the missing values to the existing Pacifica 2020 fingerprint instead of adding a second one?

**Comment by TheWizard1328** [comment]
Changes made and commited

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -40,9 +40,7 @@ class CAR:
...
..., 344: 8, 352: 8, 362: 8, 368: 8, 376: 3, 384: 8, 388: 4, 416: 7, 448: 6, 456: 4, 464: 8, 469: 8, 480: 8, 500: 8, 501: 8, 512: 8, 514: 8, 516: 7, 517: 7, 520: 8, 524: 8, 526: 6, 528: 8, 532: 8, 536: 8, 542: 8, 544: 8, 557: 8, 559: 8, 560: 8, 564: 8, 571: 3, 579: 8, 584: 8, 608: 8, 624: 8, 625: 8, 632: 8, 639: 8, 650: 8, 656: 4, 658: 6, 660: 8, 669: 3, 671: 8, 672: 8, 676: 8, 678: 8, 680: 8, 683: 8, 703: 8, 705: 8, 706: 8, 709: 8, 710: 8, 719: 8, 720: 6, 729: 5, 736: 8, 746: 5, 752: 2, 754: 8, 760: 8, 764: 8, 766: 8, 770: 8, 773: 8, 776: 8, 779: 8, 782: 8, 784: 8, 792: 8, 799: 8, 800: 8, 804: 8, 808: 8, 816: 8, 817: 8, 820: 8, 825: 2, 826: 8, 832: 8, 838: 2, 848: 8, 853: 8, 856: 4, 860: 6, 863: 8, 882: 8, 897: 8, 906: 8, 924: 8, 926: 3, 937: 8, 938: 8, 939: 8, 940: 8, 941: 8, 942: 8, 943: 8, 947: 8, 948: 8, 962: 8, 969: 4, 973: 8, 974: 5, 979: 8, 980: 8, 981: 8, 982: 8, 983: 8, 984: 8, 992: 8, 993: 7, 995: 8, 996: 8, 1000: 8, 1001: 8, 1002: 8, 1003: 8, 1008: 8, 1009: 8, 1010: 8, 1011: 8, 1012: 8, 1013: 8, 1014: 8, 1015: 8, 1024: 8, 1025: 8, 1026: 8, 1031: 8, 1033: 8, 1050: 8, 1059: 8, 1098: 8, 1100: 8, 1235: 8, 1250: 8, 1543: 8, 1875: 8, 1890: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8},
```
Comment (pd0wm) [inline selfdrive/car/chrysler/values.py:43]:
```suggestion
    {
    55: 8, 179: 8, 181: 8, 257: 5, 258: 8, 264: 8, 268: 8, 274: 2, 280: 8, 284: 8, 288: 7, 290: 6, 292: 8, 294: 8, 300: 8, 308: 8, 320: 8, 324: 8, 331: 8, 332: 8, 344: 8, 352: 8, 362: 8, 368: 8, 376: 3, 384: 8, 388: 4, 416: 7, 448: 6, 456: 4, 464: 8, 469: 8, 480: 8, 500: 8, 501: 8, 512: 8, 514: 8, 516: 7, 517: 7, 520: 8, 524: 8, 526: 6, 528: 8, 532: 8, 536: 8, 542: 8, 544: 8, 557: 8, 559: 8, 560: 8, 564: 8, 571: 3, 579: 8, 584: 8, 608: 8, 624: 8, 625: 8, 632: 8, 639: 8, 650: 8, 656: 4, 658: 6, 660: 8, 669: 3, 671: 8, 672: 8, 676: 8, 678: 8, 680: 8, 683: 8, 703: 8, 705: 8, 706: 8, 709: 8, 710: 8, 719: 8, 720: 6, 729: 5, 736: 8, 746: 5, 752: 2, 754: 8, 760: 8, 764: 8, 766: 8, 770: 8, 773: 8, 776: 8, 779: 8, 782: 8, 784: 8, 792: 8, 799: 8, 800: 8, 804: 8, 808: 8, 816: 8, 817: 8, 820: 8, 825: 2, 826: 8, 832: 8, 838: 2, 848: 8, 853: 8, 856: 4, 860: 6, 863: 8, 882: 8, 897: 8, 906: 8, 924: 8, 926: 3, 937: 8, 938: 8, 939: 8, 940: 8, 941: 8, 942: 8, 943: 8, 947: 8, 948: 8, 962: 8, 969: 4, 973: 8, 974: 5, 979: 8, 980: 8, 981: 8, 982: 8, 983: 8, 984: 8, 992: 8, 993: 7, 995: 8, 996: 8, 1000: 8, 1001: 8, 1002: 8, 1003: 8, 1008: 8, 1009: 8, 1010: 8, 1011: 8, 1012: 8, 1013: 8, 1014: 8, 1015: 8, 1024: 8, 1025: 8, 1026: 8, 1031: 8, 1033: 8, 1050: 8, 1059: 8, 1098: 8, 1100: 8, 1235: 8, 1250: 8, 1543: 8, 1875: 8, 1890: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8
    },
```

Comment (pd0wm) [reply selfdrive/car/chrysler/values.py]: Hoped this would make the diff cleaner, but it didn't.

**Comment by pd0wm** [comment]
You just replaced the original fingerprint with your own, not combined them. Your cas has `352, 362, 516, 517, 524, 526, 536, 656, 776, 1543` as extra values. Please add those.

**Comment by TheWizard1328** [comment]
Sorry I will add in the other numbers now

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by TheWizard1328** [comment]
Fixxed I hope

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by adeebshihadeh** [comment]
Several messages were removed. The new fingerprint should be a superset of the old one. Just add your new ones to the existing fingerprint.

**Comment by TheWizard1328** [comment]
That’s what I did on the final one. I think.

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by TheWizard1328** [comment]
Just checked my branch again and for some reason the values were missing again. Just updated them again

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by adeebshihadeh** [comment]
`847: 1` is still missing

**Comment by TheWizard1328** [comment]
Fixed.

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.facebook.com/WizardWorxx13>


________________________________

**Comment by adeebshihadeh** [comment]
This only adds one new message, but based on one of your routes, you'll need several messages added. Try starting over by getting the fingerprint from your current and adding it to the existing one, and ensure the combined one works on your car.

**Comment by TheWizard1328** [comment]
What do you mean.
What steps do I need to do to get these values. I have no idea what it is that you need me to get and where to put it. I only just started using my OP a few days ago and it has been a while since I’ve used C#, and Github and I have never till now used terminal in this way to access data from a device like this.

Please be patient with me I am a noob here for the most part.

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Comment by adeebshihadeh** [comment]
I updated the PR with your missing values. Pull the branch on your device and ensure your car is correctly identified.

**Comment by adeebshihadeh** [comment]
Did you confirm that this works?

**Comment by TheWizard1328** [comment]
It does been using it for a few days now.

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542<tel:(780)%20236-1542>
www.Facebook.com/WizardWorxx13<http://www.Facebook.com/WizardWorxx13>

**Pull request merged**